tests/test_api.py

```diff
@@ -14,11 +14,14 @@ def test_make_bundle(directory, expected):
         expected_bundle = yaml.safe_load(expected_file)
         assert unformat_bundle(bundle) == unformat_bundle(expected_bundle)
 
+
 @pytest.mark.parametrize('yaml_files,expected', [
-(["tests/test_files/bundles/api/bundle1/crd.yml",
-"tests/test_files/bundles/api/bundle1/csv.yaml",
-"tests/test_files/bundles/api/bundle1/packages.yaml"],
-"tests/test_files/bundles/api/bundle1/results/bundle.yaml"),
+    ([
+         "tests/test_files/bundles/api/bundle1/crd.yml",
+         "tests/test_files/bundles/api/bundle1/csv.yaml",
+         "tests/test_files/bundles/api/bundle1/packages.yaml"
+     ],
+     "tests/test_files/bundles/api/bundle1/results/bundle.yaml"),
```

Comment:
Same comment as above about limiting prs to changes related to the commit message and purpose.